packages/grafana-prometheus/src/datasource.ts

```diff
@@ -883,6 +781,18 @@ export class PrometheusDatasource
     return getOriginalMetricName(labelData);
   }
 
+  /**
+   * This converts the adhocVariableFilter array and converts it to scopeFilter array
+   * @param filters
+   */
+  generateScopeFilters(filters?: AdHocVariableFilter[]): ScopeSpecFilter[] {
```

Comment:
Nothing new just pulled a few lines down. So it is close to its counterpart `enhanceExprWithAdHocFilters`.